nautobot/dcim/templates/dcim/location.html

```diff
@@ -128,6 +128,40 @@
                 <td>{{ object.contact_email|hyperlinked_email }}</td>
             </tr>
         </table>
+        {% with request.path|add:"?tab=contacts"|urlencode as return_url %}
+            <div class="panel-footer text-right noprint">
+                {% url 'extras:map_contact_from_location' as map_url %}
+                {% with object.pk|urlencode as pk %}
+                {% with object.contact_name|urlencode as name %}
+                {% with object.contact_phone|urlencode as phone %}
+                {% with object.contact_email|urlencode as email %}
+                {% with content_type.id|urlencode as contenttypeid %}
```

Comment:
Do these have to be separate statements due to the use of `|urlencode`, or can you do something like `{% with pk=object.pk name=object.contact_name|urlencode phone=object.contact_phone|urlencode ...` as a single statement?